src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -610,6 +620,34 @@ MavlinkReceiver::handle_message_command_ack(mavlink_message_t *msg)
 }
 
 void
+MavlinkReceiver::handle_message_command_ack2(mavlink_message_t *msg)
```

Comment:
Could you please send a PR against the MAVLink devguide detailing the proposal?
https://github.com/mavlink/mavlink-devguide

Please also try to find a better name for ACK2 - it should have a notion of how ACK and the new ACK messages are different (like GPS and GPS_INT or ATTITUDE and ATTITUDE_QUAT).

Thanks!